src/fixed_width_ints.rs

```diff
@@ -92,5 +92,10 @@ cfg_if! {
 
         // static_assert_eq!(core::mem::size_of::<__uint128_t>(), _SIZE_128);
         // static_assert_eq!(core::mem::align_of::<__uint128_t>(), _ALIGN_128);
+    } else if #[cfg(all(target_arch = "aarch64", any(target_os = "macos", target_os = "ios", target_os = "tvos", target_os = "watchos")))] {
+        /// C __int128_t (alternate name for [__int128][])
+        pub type __int128_t = i128;
+        /// C __uint128_t (alternate name for [__uint128][])
+        pub type __uint128_t = u128;
```

Comment:
`__int128` and `__uint128` aren't defined in this branch of the cfg.

```suggestion
        /// C `__int128_t`
        pub type __int128_t = i128;
        /// C `__uint128_t`
        pub type __uint128_t = u128;
```

Reply:
sure